Kconfig.nrf

```diff
@@ -20,6 +20,9 @@ menu "Nordic nRF Connect"
 config ROM_START_OFFSET
 	default 0 if PARTITION_MANAGER_ENABLED # && !SYSBUILD_CONTROLLED
 
+config MCUBOOT_EXTRA_IMGTOOL_ARGS
+	default "--pad-header --header-size 0x200"
```

Comment:
this is what I had issues with and was the reason for that sdk-zephyr patch, so it used the zephyr Kconfigs instead. I guess for PM we will just have to accept the old child/parent image way it worked

Reply:
I found more issues with trying to use the ROM_START_OFFSET, cause the way partition manager calculated addresses and adjusting the linker script was causing a lot of issues as soon as 3 images for a core were included in the build.

So secure boot, mcuboot, and application built for songle core would result in growing number of issues.
Hence i'm looking at this direction, as it overall seems to requires less adjustments and can be handled without patching Zephyr.

It further gives us an indication if we should reconsider how this is done upstream with sysbuild.
Cause sysbuild could be handling more than just mcuboot + app, but imagetool will most likely only sign one image.
In such case, setting ROM_START_OFFSET on all extra images might not be ideal, if only app is signed.